telegram_bot.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, config: dict, bot, loop):
 
         first_keyboard_buttons = [
             [KeyboardButton('/daily \U0001F4B8'), KeyboardButton('/open_orders \U0001F4CB'), KeyboardButton('/position \U0001F4CC')],
-            [KeyboardButton('/closed_trades \U0000274E'), KeyboardButton('/show_config \U0001F4DD')],
+            [KeyboardButton('/closed_trades \U0000274E'), KeyboardButton('/show_config \U0001F4DD'), KeyboardButton('/set_notification \U0001F514')],
```

Comment:
I'm wondering if it would be wiser to place this button on the second page or else create a third page. My experience is that buttons that modify state are always pressed on purpose, while the first page has a tendency to get pressed accidentally. That's the reason why I moved the reload_config button to the second page of buttons recently. I think this one doesn't really have an impact if pressed accidentally, but it does fall out of line with the first page of buttons all being informative only.